data/campaigns/tutorial/scenarios/02_Tutorial_part_2.cfg

```diff
@@ -871,10 +871,9 @@ A full list of abilities and weapons specials, along with traits, may be found i
         [filter]
             side=1
             race=elf
-            level=2
-            [or]
-                level=3
-            [/or]
```

Comment:
Maybe we could support syntax like
level=2,3
(currently, the game would interpret this as level=2)